variables.tf

```
@@ -10,6 +10,11 @@ variable "os_type" {
   description = "Windows or Linux."
   type        = string
   default     = "Linux"
+
+  validation {
+    condition     = contains(["Windows", "Linux"], var.os_type)
```

Worth adding a `lower` here? Just thinking it may break some peoples implementations if they are passing `windows` instead of `Windows`.
```suggestion
    condition     = contains(["windows", "linux"], lower(var.os_type))
```

---

Added in the latest commit